report/components/core/Labels.tsx

```diff
@@ -83,12 +83,7 @@ const _AuthorLabel = ({ index }: LabelProps) => {
         );
     } else if (platform === "telegram") {
         // TODO: two letters
-        let letter: string = "";
-        // iterate UTF-8 codepoints
-        for (const symbol of author.n) {
-            letter = symbol;
-            break;
-        }
```

Comment:
This for should be kept since using [0] will return the first ASCII character and using `for .. of` will - as the comment says - get the first UTF-8 codepoint, which is not what we want.

Reply:
Check the difference between:
```js
let name = '🐛 username';

for(let i = 0; i < name.length; i++) console.log(name[i]);
for(const c of name) console.log(c);
```
One will get you � and the other 🐛.

Reply:
Strangely enough, I encounted this issue myself on my own project. I was making a typing-style animation for the title of a slideshow, and was using slicing to get each character. However, keep in mind that `for(const c of name)` doesn't solve the problem in all cases either
```js
let name = '👨‍👧‍👧 username';
for(let i = 0; i < name.length; i++) console.log(name[i]);
for(const c of name) console.log(c);
```
One you get `�� �� ��` and your fix does `👨 👧 👧` which still isn't quite right

Reply:
Yeah we don't need variations and stuff, its just an approximation :P

Reply:
Alright, fair enough haha